docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
   - Click **Continue**.
 - Click the **Save** button and wait for Strapi to restart.
 
-## 5. Manage and add data to Content Type
+## 5. Create another new Content type
+
+Navigate back to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
```

Comment:
Can you please apply the review from #3454 in this file?

Reply:
This was addressed in the other branch. So I won't touch anything here further than this comment and to reference my comments on the other branch.